kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -22,11 +25,20 @@ def __init__(self, app, port=5000, brain=None):
         self.app = app
         self.port = port
         self.brain = brain
+        self.allow_cors_request = allow_cors_request
 
         # Flask configuration remove default Flask behaviour to encode to ASCII
         self.app.url_map.strict_slashes = False
         self.app.config['JSON_AS_ASCII'] = False
 
+        if self.allow_cors_request is True:
+            try:
+              from flask_cors import CORS, cross_origin
```

Comment:
It's not native. I've done it this way so user can choose to install flask_cors or not. Either way, the code works and even log an error is the feature is activated but the lib absent.

I think only people wanting to use the API will want this additional lib.

I can change this PR to make it mandatory if needed.